### DISCUSSION TITLE
Server Migration - How to move attachments?

### DISCUSSION THREAD

**Discussion by a7maadf** [firefly-iii/firefly-iii#8064] [General]
Hello,
I spend couple of hours today migrating my Firefly III instance from a server to another, I successfully managed to migrate the database and everything is working smoothly, the only issue is with the attachments, I couldn't migrate them and ofc getting 404s whenever i try to access them.

I'm using docker-compose btw

Thank you

![image](https://github.com/firefly-iii/firefly-iii/assets/89315818/ad6a04dd-a303-48c2-9a6d-f40e5530e6b2)

**Comment by JC5** [comment]
The files are inside the volume called `firefly_iii_uploads`, and should still be present on your old location.

**Reply by a7maadf** [reply]
Thank you so much
